### ISSUE TITLE
No module named 'dbt.compat'

### ISSUE THREAD

**Issue by ghost** [jacobm001/dbt-mssql#19]
Having a bit of trouble with `dbt-mssql` installed with pip on Linux, CentOS7 using Python 3.7, running in a virtual environment. I am able to connect to sql server using vanilla python scripts without any issues.

### error
```bash
$ dbt run
Running with dbt=0.15.0
Error importing adapter: No module named 'dbt.compat'
Encountered an error while reading profiles:
  ERROR Runtime Error
  Credentials in profile "my-mssql-db", target "dev" invalid: Runtime Error
    Could not find adapter type mssql!
Defined profiles:
 - my-mssql-db
 - dbt-sqlserver
 - default
```

**Comment by ghost** [comment]
FYI: changing

```python
install_requires=[
        'dbt-core>=0.14.0',
        'pyodbc'
    ]
```
to
```python
install_requires=[
        'dbt-core==0.14.0',
        'pyodbc'
    ]
```

Appears to fix the issue.

**Comment by jacobm001** [comment]
Thank you @dougscc. I have implemented your change in the `v1.0.8` release. A `v1.1.0` will come soon to address the dbt-core 15.x branch within the next week.
